packages/discord.js/typings/index.d.ts

```diff
@@ -6273,7 +6275,7 @@ export interface MessageCreateOptions extends BaseMessageOptionsWithPoll {
   tts?: boolean;
   nonce?: string | number;
   enforceNonce?: boolean;
-  reply?: ReplyOptions;
+  messageReference?: MessageReference & { failIfNotExists?: boolean };
```

Comment:
This is gonna make this PR sem major, which I am fine with but would need to get a sem minor implementation for v14 too

Reply:
Yes, that was [discussed internally](https://canary.discord.com/channels/222078108977594368/1209960895141388389/1298915087838351422), and a v14 PR will follow this one.